angular/base/src/app/app.component.ts

```diff
@@ -6,7 +6,8 @@ import { StatusBar } from '@ionic-native/status-bar/ngx';
 
 @Component({
   selector: 'app-root',
-  templateUrl: 'app.component.html'
+  templateUrl: 'app.component.html',
+  styleUrls: ['app.component.scss']
```

Comment:
With this addition, the app.scss is no longer global, so the comment inside the file is incorrect.

@brandyscarney Didn't we have global.scss _and_ app.scss for a reason?

Reply:
Yes I believe this is why we were separating out the `app.scss` file for global styles/mixins.

Reply:
So the logic here is that global should be for global styles and mixins, and app.component.scss should be scoped styles for the app component (if any is needed)


If we have global.scss and app.scss, they both are treated at the same level. Which doesn't really make sense to keep both. I'd say just keep global.scss and then if devs want to have more, they can be the one to make that call.